apps/web/src/pages/PoolCandidates/AllPoolCandidatesPage/AllPoolCandidatesPage.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import IdentificationIcon from "@heroicons/react/24/outline/IdentificationIcon";
+
+import useRoutes from "~/hooks/useRoutes";
+import PoolCandidatesTable from "~/components/PoolCandidatesTable/PoolCandidatesTable";
+import SEO from "~/components/SEO/SEO";
+import AdminContentWrapper from "~/components/AdminContentWrapper/AdminContentWrapper";
+import adminMessages from "~/messages/adminMessages";
+import PageHeader from "~/components/PageHeader";
+
+export const AllPoolCandidatesPage = () => {
```

Comment:
I love how easy this component was to put together 😂

Reply:
It is probably our most simple page now 😛

Reply:
Alright, added the new links in [add new links to wells on dashboard](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8318/commits/a915ebb3c1c9b9c5ce3d20a3c5d892fdc8afc7dc)